Makefile

```diff
@@ -147,6 +147,13 @@ generate_history_table_migration:
 	@echo "+\n++ Generating history table migration ...\n+"
 	@docker compose $(DC_FILE) exec backend bash -c "flask generate_history_table_migration $(TABLE)"
 
+
+# Generates a migration file for the specified table
+# Usage: make generate_migration TABLE=<table_name>
+generate_table_migration:
+	@echo "+\n++ Generating history table migration ...\n+"
+	@docker compose $(DC_FILE) exec backend bash -c "flask generate_table_migration $(TABLE)"
+
```

Comment:
Added a little script to auto-generate a CREATE TABLE migration for new tables.
Pretty much the same as the history table creation scripts, just with creating the actual table instead of a `_version` table